src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -655,4 +702,23 @@ BOOST_AUTO_TEST_CASE(MishFunctionTest)
   CheckDerivativeCorrect<MishFunction>(desiredActivations,
                                         desiredDerivatives);
 }
+
+/**
+ * Basic test of the ISRU function.
+ */
+BOOST_AUTO_TEST_CASE(ISRUFunctionTest)
+{
+  const arma::colvec desiredActivations("-1.69030851 2.24928623 \
```

Comment:
If possible comment how you got these values like calculated using python interpreter or using tensor flow etc.
Not necessary though but might be helpful for anyone who goes through the test cases.

Reply:
I manually calculated them using a calculator. I thought they are not much so manual will be faster.(It wasn't)

Reply:
I think you can leave it then, resolve this conversation.

Reply:
Thanks for the fast patches.